spec/models/staff_spec.rb

```diff
@@ -65,13 +66,15 @@
     describe ".assessors" do
       subject { described_class.assessors }
 
-      context "when assessor == true" do
-        let(:assessor) { create(:staff, :assessor) }
+      context "when with_award_decline_permission == true" do
```

Comment:
```suggestion
      context "when award_decline_permission is true" do
```

Usually the contexts are written as words.